pkg/ruler/manager.go

```diff
@@ -314,6 +314,10 @@ func (r *DefaultMultiTenantManager) getOrCreateNotifier(userID string) (*notifie
 		},
 	}, log.With(r.logger, "user", userID))
 
+	if err != nil {
```

Comment:
[Nit] Why not tie the error check to the error's instantiation.
```suggestion
	if err != nil {
```